android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -153,6 +154,21 @@ public void getCurrentTime(final Integer key, final Callback callback) {
   }
 
   @ReactMethod
+  public void setSpeakerphoneOn(final Integer key, final Boolean speaker) {
+    MediaPlayer player = this.playerPool.get(key);
+    if (player != null) {
+      player.setAudioStreamType(AudioManager.STREAM_MUSIC);
+      AudioManager audioManager = (AudioManager)this.context.getSystemService(this.context.AUDIO_SERVICE);
+      audioManager.setMode(AudioManager.MODE_IN_COMMUNICATION);
+      if (speaker == true) {
```

Comment:
Please simplify line 163-167 to a oneliner to improve readability:
audioManager.setSpeakerphoneOn(speaker);